src/pages/core/entrypoints/reply.mdx

```diff
@@ -13,7 +13,8 @@ and interact with them.
 
 <Callout>TODO: Link to the actor model section once we have one</Callout>
 
-This endpoint gets invoked when the execution of an external contract you called
+This endpoint gets invoked when your contract receives some kind of message from
+the outside. For example when the execution of an external contract you called
```

Comment:
Now the first sentence seems a bit misleading. It's only invoked for submessages you add to the response of some other endpoint, after that submessage is executed. My point was just that the submessage does not have to be a `WasmMsg`, it can be any `CosmosMsg`.

Reply:
Agh, I misunderstood that. Will fix the wording